modules/distribution/src/bin/fips.bat

```diff
@@ -0,0 +1,362 @@
+
+
+@echo off
+rem ----------------------------------------------------------------------------
+rem Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+rem
+rem WSO2 LLC. licenses this file to you under the Apache License,
+rem Version 2.0 (the "License"); you may not use this file except
+rem in compliance with the License.
+rem You may obtain a copy of the License at
+rem
+rem http://www.apache.org/licenses/LICENSE-2.0
+rem
+rem Unless required by applicable law or agreed to in writing,
+rem software distributed under the License is distributed on an
+rem "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+rem KIND, either express or implied.  See the License for the
+rem specific language governing permissions and limitations
+rem under the License.
+
+set BC_FIPS_VERSION=1.0.2.3
+set BCPKIX_FIPS_VERSION=1.0.7
+set BCPROV_JDK15ON_VERSION=1.70.0.wso2v1
+set BCPKIX_JDK15ON_VERSION=1.70.0.wso2v1
```

Comment:
Are there already available BC versions in the current pack?

Reply:
These are the already existing BC versions in the pack

Reply:
Improved the script